### DISCUSSION TITLE
Clear pending comments when read

### DISCUSSION THREAD

**Discussion by twopiearr** [civitai/civitai#87] [Ideas]
If I have unread comments waiting, currently the only way to clear them is to select them from the bell menu. Since it's reasonable to assume that more than one of them is going to be in the same thread, this gets a little arduous if the thread is active. Anyway to clear these automatically when I read the thread?

**Comment by JustMaier** [comment]
If we made it so that clicking on one would clear all notifications in the same thread, does that solve this for you?

**Reply by twopiearr** [reply]
Yup!
